meta-balena-supervisor/recipes-devtools/nodejs/nodejs_8.12.0.bbappend

```diff
@@ -0,0 +1,15 @@
+DEPENDS = "openssl zlib"
+
+do_configure() {
+    rm -rf ${S}/deps/openssl
+    export LD="${CXX}"
+    GYP_DEFINES="${GYP_DEFINES}" export GYP_DEFINES
+    # $TARGET_ARCH settings don't match --dest-cpu settings
+   ./configure --prefix=${prefix} --with-intl=none --without-snapshot --shared-openssl --shared-zlib \
```

Comment:
What exactly are you changing here?

Reply:
The `--with-intl=none` part, removing internationalisation because it comes with a pretty heavy data library (around 30mb).

Reply:
I see now. Can't you overwrite it with ARCHFLAGS? It's a smaller hack. Ungly still though.

Reply:
I'm not sure, how would one go about that?

Reply:
What I'm thinking is appending to ARCHFLAGS (http://cgit.openembedded.org/meta-openembedded/tree/meta-oe/recipes-devtools/nodejs/nodejs_8.14.0.bb#n35). So basically see if this works:
ARCHFLAGS_append = " --with-intl=none"
Mind the space in front of the value string.